JAGPropertyConverterTests/JAGPropertyConverterTest.m

```diff
@@ -144,6 +172,34 @@ - (void) testIdentifyModel {
     [self assert:testModel isEqualTo:testModelDict];
 }
 
+- (void) testComposeModelWithKeypathCustomMapping {
+    NSDictionary *testModelDict = [NSDictionary dictionaryWithObjectsAndKeys:
+                                   @"G653", @"testModelID",
+                                   @"Happy", @"stringProperty",
+                                   @"MPID001", @"keypathProperty1",
+                                   @"MPID002", @"keypathProperty2",
```

Comment:
directly using keypath in dict key should also be possible (= both ways must be possible, direct or indirect)

    @(1337), @"modelProperty.intProperty",